src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -23,6 +23,16 @@
  */
 class HtmlErrorRenderer implements ErrorRendererInterface
 {
+    private const FORMATS = [
```

Comment:
let's make it public and use it in FileLinkFormatter?

Reply:
Ok, updated composer dependency constraint.

Switched name to `LINK_FORMATS`.